tensorflow/api_template.__init__.py

```diff
@@ -100,8 +99,11 @@
 if "getsitepackages" in dir(_site):
   _site_packages_dirs += _site.getsitepackages()
 
-if "sysconfig" in dir(_distutils):
-  _site_packages_dirs += [_distutils.sysconfig.get_python_lib()]
+try:
+  if "sysconfig" in dir(_distutils):
+    _site_packages_dirs += [_distutils.sysconfig.get_python_lib()]
+except NameError:
+  pass
```

Comment:
Do you have pointers on how to remove this part?

Reply:
I think it should be ok to fully remove. If testing shows that this is a problem, then let's revisit.